x-pack/auditbeat/processors/sessionmd/processdb/db.go

```diff
@@ -568,6 +566,11 @@ func setSameAsProcess(process *types.Process) {
 	}
 }
 
+func (db *DB) HasProcess(pid uint32) bool {
+	_, ok := db.processes[pid]
```

Comment:
lock here?

Reply:
I've added a lock. It was without because this is read-only, but it's still better to have, something else could write at the same time.